nixos/modules/services/misc/gitea.nix

```diff
@@ -349,7 +349,7 @@ in
         {
           DOMAIN = cfg.domain;
           STATIC_ROOT_PATH = cfg.staticRootPath;
-          LFS_JWT_SECRET = "#jwtsecret#";
+          LFS_JWT_SECRET = "#lfsjwtsecret#";
```

Comment:
Does this mean that if I don't change the secrets I would be using this well known secret? That sounds scary.

Reply:
Actually not. Have a look at the `preStart` in the systemd unit (line 470).
For each secret there is a dedicated file. If the file does not exist, a new secret will be generated and saved in this file. Afterwards every "magic string" will be replaced with the value saved in that file.

So as long as you don't mess around with these files you should be fine.

Reply:
Yeah, noticed that later, too but it is a bit complicated and can be easily miss understood.